tests/manage/cluster/test_remove_mon_from_cluster.py

```diff
@@ -58,6 +54,7 @@ def run_io_on_pool(pool_obj):
 
 
 @tier4
+@ignore_leftovers
```

Comment:
please remove this

Reply:
Because a mon deployment is deleted and a new deployment is created with a new letter that didn't exist before the test tier down is failing for leftovers.

Reply:
The environment checker for leftovers should take care of such a scenario.
Could you please execute and paste the output of the leftovers?

Reply:
E           ocs_ci.ocs.exceptions.ResourceLeftoversException: 
E           There are leftovers in the environment after test case:
E           Resources added:
E           - '***pods***':
E             - apiVersion: v1
E               kind: Pod
E               metadata:
E                 annotations:
E                   openshift.io/scc: rook-ceph
E                 creationTimestamp: '2019-08-13T11:17:28Z'
E                 generateName: rook-ceph-mon-l-86f58d777f-
E                 labels:
E                   app: rook-ceph-mon
E                   ceph_daemon_id: l
E                   mon: l
E                   mon_cluster: openshift-storage
E                   pod-template-hash: 86f58d777f
E                   rook_cluster: openshift-storage
E                 name: rook-ceph-mon-l-86f58d777f-9c898
E                 namespace: openshift-storage
E                 ownerReferences:
E                 - apiVersion: apps/v1
E                   blockOwnerDeletion: true
E                   controller: true
E                   kind: ReplicaSet
E                   name: rook-ceph-mon-l-86f58d777f
E                   uid: efa56f4a-bdbb-11e9-bcc0-0641c477f880
E                 resourceVersion: '1411406'
E                 selfLink: /api/v1/namespaces/openshift-storage/pods/rook-ceph-mon-l-86f58d777f-9c898
E                 uid: efae9dd3-bdbb-11e9-bcc0-0641c477f880

Reply:
Here is the whole output:
http://pastebin.test.redhat.com/788271

Reply:
ok that's because you are deleting deployment